{{cookiecutter.project_slug}}/requirements/local.txt

```diff
@@ -13,6 +13,7 @@ watchfiles==0.21.0  # https://github.com/samuelcolvin/watchfiles
 
 # Testing
 # ------------------------------------------------------------------------------
+ruff==0.2.1
```

Comment:
Shouldn't this stay in the code quality section, where the other ones were?

Reply:
You are right, fixed